accounts/README.md

```diff
@@ -1,10 +1,121 @@
+
+### Expected interface per Safe{Core} Protocol Module
+
+#### Plugins
+
+The account MUST implement the following interface:
+
+```Solidity
+interface IAccount {
+    function execTransactionFromModuleReturnData(
+        address to,
+        uint256 value,
+        bytes memory data,
+        uint8 operation
+    ) external returns (bool success, bytes memory returnData);
+}
+```
+
+where:
+
+| Type         | Name      | Description                                                                                                                                                                                                                                                                                                                                                                               |
+|--------------|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| address      | to        | The address of the contract to be called                                                                                                                                                                                                                                                                                                                                                  |
+| uint256      | value     | The amount of Native Token to be sent                                                                                                                                                                                                                                                                                                                                                     |
+| bytes memory | data      | The call data                                                                                                                                                                                                                                                                                                                                                                             |
+| uint8        | operation | The operation type to be executed. `0` for CALL, `1` for DELEGATECALL. The DELEGATECALL operation can only be executed by a Plugin with root access permissions. More information about transaction types can be found in the [Manager contract](https://github.com/safe-global/safe-core-protocol-specs/blob/2bffd759dd12be5583594f302d97c35e0ab9fcf5/manager/README.md) specifications. |
```

Comment:
nit: We should use a relative link here so that it is stays up to date:

```suggestion
| uint8        | operation | The operation type to be executed. `0` for CALL, `1` for DELEGATECALL. The DELEGATECALL operation can only be executed by a Plugin with root access permissions. More information about transaction types can be found in the [Manager contract](../manager/README.md) specifications. |
```